modules/tinymce/src/themes/silver/test/ts/browser/window/SilverDialogBlockTest.ts

```diff
@@ -140,6 +143,27 @@ describe('browser.tinymce.themes.silver.window.SilverDialogBlockTest', () => {
         DialogUtils.close(editor);
         I18n.setCode('en');
       });
+
+      it('TINY-10056: Ensure close button is clickable when dialog is blocked', async () => {
+        const editor = hook.editor();
+        const api = DialogUtils.open(editor, dialogSpec, test.params);
+        api.block('Block message');
+        await pClick(editor, closeButtonSelector);
+        store.cAssertEq(`Ensure that it has clicked (${test.label})`, [ 'clicked' ]);
```

Comment:
I think we should sanity check the dialog actually closed
```suggestion
        store.cAssertEq(`Ensure that it has clicked (${test.label})`, [ 'clicked' ]);
          UiFinder.notExists(SugarBody.body(), 'div[role=dialog]');
```